tools/compose-image-yml.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
```

Comment:
It's generally better to avoid `-e` flag. My bad I did not check if command executed successfully. Any chance you can adjust pull request with checking the exit code instead of `-e`?

Reply:
Modified. It would be nice to add some error handling logic inside, probably in future PRs.